README.markdown

```diff
@@ -1,15 +1,63 @@
 semigroupoids
-==========
+=============
 
 [![Hackage](https://img.shields.io/hackage/v/semigroupoids.svg)](https://hackage.haskell.org/package/semigroupoids) [![Build Status](https://github.com/ekmett/semigroupoids/workflows/Haskell-CI/badge.svg)](https://github.com/ekmett/semigroupoids/actions?query=workflow%3AHaskell-CI)
 
-A semigroupoid is a `Category` without `id`.
+A semigroupoid is a `Category` without `id`. This package provides a range of 
+`id`-free versions of type classes, as well as some supporting functions and
+data types.
+
+Field Guide
+-----------
+
+The diagram below describes the relationships between the type classes defined
+in this package, and those from `base` (with some from `contravariant` as well). Thick-bordered
+nodes correspond to type classes defined in this package; thin-bordered ones are
+from elsewhere. Solid edges represent subclass relationships that actually
+exist; dashed edges are those which _should_ exist in theory.
+
+![A diagram of the relationships between type classes defined in this package and elsewhere.](https://raw.githubusercontent.com/ekmett/semigroupoids/master/img/classes.svg)
```

Comment:
I thought this would render the SVG image inline, but looking [here](https://github.com/kozross/semigroupoids/tree/05580a52aa777267047ca205b2994f8a19d41942#field-guide), that doesn't seem to be the case. [This StackOverflow answer](https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown/16462143#16462143) suggests I got the specifics wrong. Can you try this instead?

```suggestion
![A diagram of the relationships between type classes defined in this package and elsewhere.](https://raw.github.com/ekmett/semigroupoids/master/img/classes.svg)
```

Reply:
The other issue, now that I think about it more closely, is that `img/classes.svg` isn't on the `master` branch yet, so of _course_ it wouldn't render an image that refers to something in the `master` branch. Silly me.

In any case, I'm pretty sure it's correct now, so I'll just merge it and fix it up _post facto_ if need be.